export/mlab_export_test.py

```diff
@@ -0,0 +1,440 @@
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit tests for collectd-mlab."""
+
+import logging
+import os
+import StringIO
+import unittest
+
+# Third-party modules.
+import mock
+
+# Module under test.
+import mlab_export
+
+# Too many public methods. Hard to avoid for unit tests.
+# pylint: disable=R0904
+
+# Static timestamp for unittests.
+FAKE_TIMESTAMP = 1410001000
```

Comment:
Suggest mentioning the human-readable equivalent in a comment since we check this value against a YYYYMMDD value in some tests.